korge/src/commonMain/kotlin/com/soywiz/korge3d/BitmapFontExt3D.kt

```diff
@@ -0,0 +1,89 @@
+package com.soywiz.korge3d
+
+import com.soywiz.kds.get
+import com.soywiz.kds.iterators.fastForEach
+import com.soywiz.kds.iterators.fastForEachWithIndex
+import com.soywiz.kmem.clamp
+import com.soywiz.korag.AG
+import com.soywiz.korge.view.BlendMode
+import com.soywiz.korim.color.Colors
+import com.soywiz.korim.color.RGBA
+import com.soywiz.korim.font.BitmapFont
+import com.soywiz.korma.geom.Matrix
+import com.soywiz.korma.geom.Vector3D
+import com.soywiz.korma.geom.invert
+
+fun BitmapFont.drawText3D(
+    ctx: RenderContext3D,
+    v1: Vector3D,
+    v2: Vector3D,
+    v3: Vector3D,
+    v4: Vector3D,
```

Comment:
What is each vector for? Either a small documentation line explaining them, or if there are other names than vN to understand these parameters would be nice

Reply:
As I checked later, this might be the face containing the text? Maybe we can create a class for the Face? Also, could we determine a quad in the 3D space with only three points and compute the other one? Maybe a Face class could allow to construct that instance several ways and provide it here.

Reply:
good idea. 
see above comment re 3D Text being in progress

Reply:
As you prefer. If you think this PR without the text additions makes sense already, we can merge it, so you can add the text one in a separate PR.

Reply:
the 3D Text is proving to be more work than I initially though, so it is probably best to keep that to a diff PR.
If it is easy for you to pick out the shiness typo fixes and MeshBuilder stuff, then do that from this PR.
Else close it and I will try and create a diff branch without the 3D Text bits, and give you a new PR from that.

Reply:
It's Ok. This repo doesn't need to be super stable right now. I can merge as it is right now, so we don't have to rebase stuff

Reply:
ok. great. I am will continue the 3D Text, and give a separate PR when I have something (more) working.